crypto/keyring/keyring.go

```diff
@@ -401,12 +402,23 @@ func (ks keystore) SaveLedgerKey(uid string, algo SignatureAlgo, hrp string, coi
 	if err != nil {
 		return nil, err
 	}
-
-	return ks.writeLedgerKey(uid, priv.PubKey(), *hdPath, algo.Name())
```

Comment:
I assume, I dont deal with algo in SaveLedgerKey. Since this field is in pubkey, i can remove algo argument from SaveLedgerKey right?

Reply:
Yes

Reply:
So, I would keep it.